layers/+source-control/version-control/README.org

```diff
@@ -25,11 +28,31 @@ To use this contribution add it to your =~/.spacemacs=
 (setq-default dotspacemacs-configuration-layers '(version-control))
 #+END_SRC
 
+Additionally, you can choose the package to facilitate the diff micro-state and
+show margins by setting =version-control-diff-tool=
+
+#+BEGIN_SRC emacs-lisp
+(setq-default dotspacemacs-configuration-layers '(version-control :variables
+                                                                  version-control-diff-tool 'diff-hl))
+#+END_SRC
+
+
 * Key Bindings
 
-| Key Binding | Description   |
-|-------------+---------------|
-| ~SPC g d g~ | Go to hunk    |
-| ~SPC g d N~ | Previous hunk |
-| ~SPC g d n~ | Next hunk     |
-| ~SPC g d r~ | Revert hunk   |
+| Key Binding | Description               |
+|-------------+---------------------------|
+| ~SPC g .~   | vcs hunk micro-state      |
+
+VCS Hunk Micro-state
+
+| Key Binding | Description                   |
```

Comment:
Horizontal line missing in this table, it's needed for `Key Binding | Description` to be shown as a header.